client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java

```diff
@@ -55,7 +55,7 @@
  * @author Mircea.Markus@jboss.com
  * @since 4.1
  */
-public class RemoteCacheManager implements BasicCacheContainer {
```

Comment:
Btw, we had a very similar issue with org.infinispan.commons.util.FileLookup. It used to be an interface and we let it become a class in 7.0, and in 7.1 (and 7.0.3) we had to revert that back so that other projects could easily use different Infinispan versions without having to recompile them.